assessments/projects/volcano/threat-modelling.md

```diff
@@ -0,0 +1,59 @@
+# Lightweight Threat Modelling
```

Comment:
I'd be careful with the name as these notes are more akin to a lightweight threat analysis/enumeration as opposed to threat modeling which represents a comprehensive exercise (see: [OWASP Threat Modeling Process](https://owasp.org/www-community/Threat_Modeling_Process)).

Reply:
I agree, changed it to threat analysis instead.